pkg/policy/validate.go

```diff
@@ -402,13 +395,13 @@ func Validate(policy kyvernov1.PolicyInterface, client dclient.Interface, mock b
 		}
 	}
 
-	if spec.SchemaValidation == nil || *spec.SchemaValidation {
+	if !mock && (spec.SchemaValidation == nil || *spec.SchemaValidation) {
```

Comment:
@vyankyGH @prateekpandey14 @realshuting does this look correct ?
I had to exclude this validation when running cli test command.

Reply:
Seems correct, fetching schema isn't available in CLI.